internal/core/core.go

```diff
@@ -189,6 +192,32 @@ func (d *Core) GetTopTags(limit int) ([]string, error) {
 	return tags, nil
 }
 
+type Project struct {
```

Comment:
We should move this to `internal/models` and embed `v1.Project` (v1 is alias for schema) like `ManifestData` in models.

Reply:
Moved it